server/routes/appointments/create-and-edit/handlers/startJourney.ts

```diff
@@ -54,7 +54,7 @@ export default class StartJourneyRoutes {
       appointments: [],
     }
     req.session.returnTo = null
-    res.redirect('upload-by-csv')
```

Comment:
I think that the `upload-by-csv` route and corresponding view is no longer used following this change. If that's the case, could you delete the UploadByCSV class, tests, references to the class and the upload-prisoners-by-csv.njk view used by that handler?

Reply:
done